### ISSUE TITLE
Small corrections in ormqr! error messages

### ISSUE THREAD

**Issue by andreasvarga** [DynareJulia/FastLapackInterface.jl#37]
I think the following lines in` ormqr!`

```
            if side == 'L' && m != mA
                throw(DimensionMismatch("for a left-sided multiplication, the first dimension of C, $m, must equal the second dimension of A, $mA"))
            end
            if side == 'R' && n != mA
                throw(DimensionMismatch("for a right-sided multiplication, the second dimension of C, $m, must equal the second dimension of A, $mA"))
            end

```
should be

```
            if side == 'L' && m != mA
                throw(DimensionMismatch("for a left-sided multiplication, the first dimension of C, $m, must equal the first dimension of A, $mA"))
            end
            if side == 'R' && n != mA
                throw(DimensionMismatch("for a right-sided multiplication, the second dimension of C, $m, must equal the first dimension of A, $mA"))
            end

```

Similar modifications are necessary also in the function 
`Base.resize!(ormws::QROrmWs, side::AbstractChar, trans::AbstractChar, ...`

and also in the original LAPACK wrapper for `ormqr!`.

**Comment by MichelJuillard** [comment]
I think that you are right. Thanks

**Comment by MichelJuillard** [comment]
Fixed in commit 37c14d922fff222f63e04a7243d565e4e855d9b5

**Comment by MichelJuillard** [comment]
Fixed in commit 37c14d922fff222f63e04a7243d565e4e855d9b5
